src/contract_wrappers/modules/transfer_manager/__tests__/general_transfer_manager_wrapper.test.ts

```diff
@@ -525,6 +540,856 @@ describe('GeneralTransferManagerWrapper', () => {
+
+      // Result expectation
+      expect(result).toBe(expectedResult);
+
+      // Verifications
+      verify(mockedContract.getInvestors).once();
+      verify(
+        mockedMethod.callAsync(
+          objectContaining(new BigNumber(mockedParams.fromIndex)),
+          objectContaining(new BigNumber(mockedParams.toIndex)),
+        ),
+      ).once();
+    });
+  });
+
+  describe('GetPermissions', () => {
+    test('should getPermissions', async () => {
+      // Address expected
+      const expectedResult = [stringToBytes32(Perm.Admin)];
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.getPermissions).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync()).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.getPermissions();
+
+      // Result expectation
+      expect(stringToBytes32(result[0])).toBe(expectedResult[0]);
+
+      // Verifications
+      verify(mockedContract.getPermissions).once();
+      verify(mockedMethod.callAsync()).once();
+    });
+  });
+
+  describe('GetAddressBytes32', () => {
+    test('should getAddressBytes32', async () => {
+      // Address expected
+      const expectedResult = stringToBytes32('0x123');
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.getAddressBytes32).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync()).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.getAddressBytes32();
+
+      // Result expectation
+      expect(stringToBytes32(result)).toBe(expectedResult);
+
+      // Verifications
+      verify(mockedContract.getAddressBytes32).once();
+      verify(mockedMethod.callAsync()).once();
+    });
+  });
+
+  describe('GetTokensByPartition', () => {
+    test('should getTokensByPartition', async () => {
+      const expectedSecurityTokenAddress = '0x3333333333333333333333333333333333333333';
+      const mockedGetSecurityTokenAddressMethod = mock(MockedCallMethod);
+      when(mockedContract.securityToken).thenReturn(instance(mockedGetSecurityTokenAddressMethod));
+      when(mockedGetSecurityTokenAddressMethod.callAsync()).thenResolve(expectedSecurityTokenAddress);
+      when(mockedContractFactory.getSecurityTokenContract(expectedSecurityTokenAddress)).thenResolve(
+        instance(mockedSecurityTokenContract),
+      );
+      const expectedDecimalsResult = new BigNumber(18);
+      const mockedSecurityTokenDecimalsMethod = mock(MockedCallMethod);
+      when(mockedSecurityTokenDecimalsMethod.callAsync()).thenResolve(expectedDecimalsResult);
+      when(mockedSecurityTokenContract.decimals).thenReturn(instance(mockedSecurityTokenDecimalsMethod));
+
+      const mockedParams = {
+        partition: "UNLOCKED",
```

Comment:
Maybe this parameter should be Partition.Unlocked

Reply:
it must be :)